### PR TITLE
HPCC-22471 Additional translations for 7.4.x

### PR DESCRIPTION
**Pull request by kunalaswani**
Missing translations for DataPatterns and MaximizeRestore added.

Signed-off-by: Kunal Aswani <Kunal.Aswani@lexisnexisrisk.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [x] This change is a new feature (non-breaking change which adds functionality).
- [ ] This change improves the code (refactor or other change that does not change the functionality)
- [ ] This change fixes warnings (the fix does not alter the functionality or the generated code)
- [ ] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)

## Checklist:
- [x] My code follows the code style of this project.
  - [ ] My code does not create any new warnings from compiler, build system, or lint.
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit is signed.
- [ ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [ ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x] I have read the CONTRIBUTORS document.
- [x] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [x] I have given due consideration to all of the following potential concerns:
  - [ ] Scalability
  - [ ] Performance
  - [ ] Security
  - [ ] Thread-safety
  - [ ] Premature optimization
  - [ ] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [ ] The target branch of this pull request is appropriate for such a change.
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately
- [x] This is a user interface / front-end modification
  - [ ] I have tested my changes in multiple modern browsers
  - [ ] The component(s) render as expected

## Smoketest:
- [ ] Send notifications about my Pull Request position in Smoketest queue.
- [ ] Test my draft Pull Request.

### CODE REVIEW

**Comment by kunalaswani** [comment]
Please review
Spanish (es): @rpastrana 
Hungarian (hu): @AttilaVamos 
Serbian, Bosnian, Croation (sr, bs, hr): @emuharemagic 
Brazil Portuguese: @dehilsterlexis 
Chinese (zh): @wangkx / @xwang2713 

Thank you

**Review by emuharemagic — APPROVED** [review]
sr,bs,hr looks good

File: esp/src/eclwatch/nls/es/hpcc.js [hunk 1]
```diff
@@ -132,6 +132,8 @@ define(
     Dali: "Dali",
     DaliIP: "IP de Dali",
     DataPatterns: "Patrones de datos",
+    DataPatternsNotStarted: "Análisis no esta encontrado. Para comenzar, oprime el botón Analizar arriba.",
```
Comment (rpastrana) [inline esp/src/eclwatch/nls/es/hpcc.js]: Análisis no se encuntra. Para comenzar, oprima el botón Analizar encima.

File: esp/src/eclwatch/nls/es/hpcc.js [hunk 2]
```diff
@@ -132,6 +132,8 @@ define(
...
+    DataPatternsStarted: "Analizando. Cuando el reporte completo se mostrará aquí.",
```
Comment (rpastrana) [inline esp/src/eclwatch/nls/es/hpcc.js]: Analizando. Cuando el reporte este completo se mostrará aquí.

**Review by rpastrana — CHANGES_REQUESTED** [review]
@kunalaswani a couple of comments

File: esp/src/eclwatch/nls/es/hpcc.js (hunk 1 above)
Comment (kunalaswani) [reply esp/src/eclwatch/nls/es/hpcc.js]: changes made

File: esp/src/eclwatch/nls/es/hpcc.js (hunk 2 above)
Comment (kunalaswani) [reply esp/src/eclwatch/nls/es/hpcc.js]: changes made

**Comment by dehilsterlexis** [comment]
@kunalaswani pt-br looks good

File: esp/src/eclwatch/nls/es/hpcc.js (hunk 2 above)
Comment (rpastrana) [reply esp/src/eclwatch/nls/es/hpcc.js]: @kunalaswani approved

**Review by rpastrana — APPROVED** [review]

**Comment by AttilaVamos** [comment]
@kunalaswani hu looks good.

**Pull request merged**